pkg/pipelinerun/description/description.go

```diff
@@ -152,6 +154,19 @@ func PrintPipelineRunDescription(s *cli.Stream, prName string, p cli.Params) err
 		return fmt.Errorf("failed to find pipelinerun %q", prName)
 	}
 
+	if pr.Spec.PipelineRef != nil {
+		pl, _ := pipeline.Get(cs, pr.Spec.PipelineRef.Name, metav1.GetOptions{}, p.Namespace())
+		//if err != nil {
+		//	return err
+		//}
+
+		if pl != nil && len(pl.Spec.Params) > 0 {
+			if err := description.SetDefault(&pr.Spec.Params, pl.Spec.Params); err != nil {
+				return err
+			}
+		}
+	}
```

Comment:
@pradeepitm12  I think this part should be reviewed more in strict way because I remove error part.

The reason for this is that if I get `err` and return err if there is, then `go test` failed because pipleline resource does not exist. ( I dived deep this problem but I could solve this yet..) 


```
pl, _ := pipeline.Get(cs, pr.Spec.PipelineRef.Name, metav1.GetOptions{}, p.Namespace())
```
So, I skipped err and if there is an error, it will skip this part (this only occurs in the test environment as far as I know). If there is no error, then SetDefault() function will be called. 

I would like to ask which is the most suitable way to make this part for `pipelinerun`

I've tested as many cases as possible, but with the real manifest file, this works well